rllib/__init__.py

```diff
@@ -57,7 +57,7 @@ def setup(self, config):
 
 _setup_logger()
 
-usage_lib.record_library_usage("rllib")
+# usage_lib.record_library_usage("rllib")
```

Comment:
Could you please explain this? Can we remove it altogether?

Reply:
Oh, sorry, this shouldn't be here. Will remove ...